cmd/create/machinepool/nodepool.go

```diff
@@ -386,6 +386,44 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		npBuilder.TuningConfigs(inputTuningConfig...)
 	}
 
+	var inputKubeletConfigs []string
```

Comment:
We have an active task on-going to refactor the `create machinepool` command to align with the runner model. I'm following the existing implementation at the minute to avoid do a large refactor in this PR.